components/eam/src/physics/cam/misc_diagnostics.F90

```diff
@@ -391,7 +391,7 @@ subroutine dcape_diags_init( pbuf, pver )
    integer,  pointer :: iptr1d(:)
 
    call pbuf_get_field( pbuf, pbuf_get_index('CAPE_old_dCAPEd'), ptr1d ); ptr1d = 0._r8 
-   call pbuf_get_field( pbuf, pbuf_get_index('Q_mx_old_dCAPEd'), ptr1d ); ptr1d = 5.e-3
+   call pbuf_get_field( pbuf, pbuf_get_index('Q_mx_old_dCAPEd'), ptr1d ); ptr1d = 5.e-3_r8
    call pbuf_get_field( pbuf, pbuf_get_index('T_mx_old_dCAPEd'), ptr1d ); ptr1d = 273._r8
    call pbuf_get_field( pbuf, pbuf_get_index(  'mx_old_dCAPEd'), iptr1d); iptr1d= pver
 
```

Comment:
@huiwanpnnl , are  the text following semicolon meant for comment? I never used it for Fortran codes. The compiler will complain about it.

Reply:
Semicolons are used here to have two statements in one line, just to make the code look a bit more compact. (It's a personal style thing :-) )

Out of curiosity, did a compiler give error messages about this?

Reply:
I see. Thanks Hui. I only used it for C and shell style codes but never Fortran. Just tested a simple code, compile by both intel and gnu compiler. They are ok. Apparently these have been in your codes  and working well.

Reply:
F90 allowed semicolons as separator for multiple statements on a line (something I just learned).

Reply:
Thank you both, @wlin7 and @rljacob, for the feedback and confirmation! 

(Every now and then I find myself using semicolons to get lines aligned so that I can check consistency more easily :-) )